pkg/alertmanager/sender.go

```diff
@@ -0,0 +1,118 @@
+// SPDX-License-Identifier: AGPL-3.0-only
```

Comment:
Out of curiosity, couldn't we technically copy this directly into grafana/alerting? With that we could have both Grafana and Mimir use the same code as opposed to a copy that could potentially diverge?

Reply:
This code is not in the alerting package. The interface is defined there, bt the implementations is up to each importing package.